src/extension.ts

```diff
@@ -67,14 +67,15 @@ export function activate(context: vscode.ExtensionContext) {
         }).map(s => {
             return {
                 label: getShellLabel(s),
-                description: getShellDescription(s)
+                description: getShellDescription(s),
+                _shell: s
```

Comment:
Could we strongly type this by creating an interface that extends QuickPickItem?

Reply:
Done.